WeatherApp/Model/WeatherResponse.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  WeatherResponse.swift
+//  WeatherApp
+//
+//  Created on 25/01/2020.
+//  Copyright © 2020 Marcin Wójciak. All rights reserved.
+//
+
+import Foundation
+
+struct WeatherResponse: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case city
+        case dayForecasts = "list"
+    }
+
+    let city: CityInfoResponse
+    let dayForecasts: [DayForecastResponse]
+
+    init(from decoder: Decoder) throws {
+        do {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+            city = try container.decode(CityInfoResponse.self, forKey: .city)
```

Comment:
Teraz z kontenera wyciągamy wartość dla klucza `.city` i mapujemy (dekodujemy JSON) na typ `CityInfoResponse`, który to implementuje protokół `Decodable`